tests/integration/__snapshots__/authorizations.test.ts.snap

```diff
@@ -196,86 +27,7 @@ exports[`authorizations should return authorisations filtered by authorization t
-        "expiry": null,
-        "fromDid": "0x0500000000000000000000000000000000000000000000000000000000000000",
-        "status": "Consumed",
-        "toDid": "0xeb686d5391c1123fcd252419bc1e77b525f46bddc8964d34e74be0988c419fea",
-        "toKey": null,
-        "type": "BecomeAgent",
-        "updatedBlock": 312,
-      },
-      Object {
-        "__typename": "Authorization",
-        "authId": 231,
-        "createdBlock": 313,
-        "data": "{\\"0x31325449434b455200000000\\",\\"{\\\\\\"full\\\\\\":null}\\"}",
-        "expiry": null,
-        "fromDid": "0x0500000000000000000000000000000000000000000000000000000000000000",
-        "status": "Consumed",
-        "toDid": "0x0854afeca045161ef42ba9be94c973f65efc0e8532933500865a1fd655148f6c",
-        "toKey": null,
-        "type": "BecomeAgent",
-        "updatedBlock": 314,
-      },
-      Object {
-        "__typename": "Authorization",
-        "authId": 232,
-        "createdBlock": 317,
-        "data": "{\\"0x31325449434b455200000000\\",\\"{\\\\\\"full\\\\\\":null}\\"}",
-        "expiry": null,
-        "fromDid": "0xeb686d5391c1123fcd252419bc1e77b525f46bddc8964d34e74be0988c419fea",
-        "status": "Consumed",
-        "toDid": "0x0500000000000000000000000000000000000000000000000000000000000000",
-        "toKey": null,
-        "type": "BecomeAgent",
-        "updatedBlock": 318,
-      },
-      Object {
-        "__typename": "Authorization",
-        "authId": 77,
-        "createdBlock": 104,
-        "data": "5DqZyPx9NAVzWhzRLeptzz9u1GwxsX5AQB3pzrUAHFFBJHCM",
-        "expiry": null,
-        "fromDid": "0x0500000000000000000000000000000000000000000000000000000000000000",
-        "status": "Pending",
-        "toDid": "0x139dad9675f6d89bbe39d4a1ffcb824b9ff44fce366390090752a926d7c6e689",
-        "toKey": null,
-        "type": "AddMultiSigSigner",
-        "updatedBlock": 104,
-      },
-      Object {
-        "__typename": "Authorization",
-        "authId": 78,
-        "createdBlock": 104,
-        "data": "5DqZyPx9NAVzWhzRLeptzz9u1GwxsX5AQB3pzrUAHFFBJHCM",
-        "expiry": null,
-        "fromDid": "0x0500000000000000000000000000000000000000000000000000000000000000",
-        "status": "Pending",
-        "toDid": "0x81c9b01d440ae8b2330e400411564d64446502b1452cc31de731730e411a6e39",
-        "toKey": null,
-        "type": "AddMultiSigSigner",
-        "updatedBlock": 104,
-      },
-      Object {
-        "__typename": "Authorization",
-        "authId": 79,
-        "createdBlock": 104,
-        "data": "5DqZyPx9NAVzWhzRLeptzz9u1GwxsX5AQB3pzrUAHFFBJHCM",
-        "expiry": null,
-        "fromDid": "0x0500000000000000000000000000000000000000000000000000000000000000",
-        "status": "Pending",
-        "toDid": "0xee7cd3d29832c3ad6db1dedb99b8fd146b57c6458fd79d17c5ac81f3d098227a",
-        "toKey": null,
-        "type": "AddMultiSigSigner",
-        "updatedBlock": 104,
-      },
-    ],
+    "nodes": Array [],
```

Comment:
This shouldn't be an empty array, it probably means something is wrong here.

Reply:
Can you once run the tests in your machine once? CI also has passed these so I'm not sure what to do here. Maybe the snapshot needs to be updated or something in `polymesh-local`

Reply:
I'll give it a shot.

With snapshot based tests passing a `-u` will make them almost always pass a CI run, the value comes in looking at the diff.